<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbWindow</class>
 <widget class="QDialog" name="DbWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QPushButton" name="buttonFrame">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStep">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReset">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableVideoRegisters">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>121</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableOtherRegisters">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>121</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDisassembler">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableMemory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>231</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>81</width>
     <height>181</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="tableRegisters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>67</width>
      <height>153</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="rowCount">
     <number>6</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>19</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>22</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <row>
     <property name="text">
      <string>PC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>AF</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BC</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DE</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>HL</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SP</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>FFFF</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>FFFF</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>FFFF</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>FFFF</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>FFFF</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>FFFF</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Registers</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
